src/log.js

```diff
@@ -198,20 +200,13 @@ class Log extends GSet {
    * @return {Log}   New Log containing the appended value
    */
   async append (data, pointerCount = 1) {
-    // Verify that we're allowed to append
-    if ((this._key && this._key.getPublic)
-        && !this._keys.includes(this._key.getPublic('hex')) 
-        && !this._keys.includes('*')) {
-      throw new Error("Not allowed to write")
-    }
-
     // Update the clock (find the latest clock)
     const newTime = Math.max(this.clock.time, this.heads.reduce(maxClockTimeReducer, 0)) + 1
     this._clock = new Clock(this.clock.id, newTime)
     // Get the required amount of hashes to next entries (as per current state of the log)
     const nexts = Object.keys(this.traverse(this.heads, pointerCount))
     // Create the entry and add it to the internal cache
-    const entry = await Entry.create(this._storage, this._keystore, this.id, data, nexts, this.clock, this._key)
+    const entry = await Entry.create(this._storage, this._acl, this.id, data, nexts, this.clock)
```

Comment:
Instead of passing the whole ACL object here, the `Entry.create` method could be broken up into create/sign/put. It might make it easier to understand how it actually works – since one might assume this is simply creating an entry instance – and then we can pass just the public key to be used as a fallback for the `clockId` property. Maybe a TODO for another PR?